packages/core/src/agent/__tests__/AgentMessage.test.ts

```diff
@@ -65,7 +65,7 @@ describe('AgentMessage', () => {
         '@type': 'https://didcomm.org/fake-protocol/2.0/message',
       }
 
-      const message = JsonTransformer.fromJSON(json, CustomProtocolMessage)
+      const message = JsonTransformer.fromJSON(json, CustomProtocolMessage, { validate: false })
 
       await expect(MessageValidator.validate(message)).rejects.toMatchObject([
         {
```

Comment:
Maybe we can call it like this (to stay consistent with how you updated it with the other tests):

```suggestion
      await expect(JsonTransformer.fromJSON(json, CustomProtocolMessage)).rejects.toMatchObject([
        {
```